packages/ember/src/components/mktg/footer.gts

```diff
@@ -0,0 +1,80 @@
+import Component from '@glimmer/component';
+import { or } from 'ember-truth-helpers';
+
+interface FooterSectionSignature {
+  Element: HTMLDivElement;
+  Args: {
+    isCollapsible?: boolean;
+  };
+  Blocks: {
+    default?: [];
+  };
+}
+
+interface MarketingFooterSignature {
+  Element: HTMLElement;
+  Args: {
+    hasHorizontalLine?: boolean;
+  };
+  Blocks: {
+    nav: [];
+    'social-media': [];
+    brand: [];
+    legal: [];
+    default?: [];
+  };
+}
+
+class FooterSection extends Component<FooterSectionSignature> {
+  get isCollapsible() {
+    return this.args.isCollapsible ?? true;
+  }
+
+  <template>
+    <div
+      class="col row
+        {{if this.isCollapsible 'row-cols-1 row-cols-md-auto' 'row-cols-auto'}}
+        align-items-center gx-3 gy-4 gx-md-4 my-0"
+    >
+      {{yield}}
+    </div>
+  </template>
+}
+
+export default class MarketingFooterComponent extends Component<MarketingFooterSignature> {
+  get hasHorizontalLine() {
+    return this.args.hasHorizontalLine ?? false;
+  }
```

Comment:
Also, if the default value is falsey, you can remove the getter entirely and convert this to a template-only component.